core/dbio/database/database_proton.go

```diff
@@ -232,6 +232,7 @@ func (conn *ProtonConn) BulkImportStream(tableFName string, ds *iop.Datastream)
 				// Do insert
 				ds.Context.Lock()
 				_, err := stmt.Exec(row...)
+				time.Sleep(1 * time.Millisecond)
```

Comment:
@jovezhong can you share why you put sleep here?

Reply:
I have not found a good reason yet, but I don't put this 1ms sleep, the count will be 0 and mismatch with the num of rows in the source table. This is a working solution and won't slow down the ingestion too much and I will try other values/solutions later.